### PR TITLE
Fix install script to work on Windows Git bash

### PR DESCRIPTION
**Pull request by beppler**
This PR allow the install script to be runned on Windows using Git bash as it happens on gobrew github action.

After merging this it probably will be possible to enable the current actions to run on windows-latest too.

### CODE REVIEW

**Review by kevincobain2000 — APPROVED** [review]

**Comment by kevincobain2000** [comment]
Will merge after @juev's approval

**Review by juev — APPROVED** [review]
After installation we print help message from gobrew. I think that's enough.

**Comment by kevincobain2000** [comment]
Merged! Hopefully now action on windows can work and we can dominate the world by storm!

**Pull request merged**
